images/entrypoint.sh

```diff
@@ -21,7 +21,7 @@
 # OA4MP is only for origin. Don't configure it if the deamon is not origin
 # the script is expect the argument in "[osdf|pelican] [deamon name] [...args]"
 if [ "$2" == "origin" ]; then
-    echo "Initializing run environment for Pelican..."
+    #echo "Initializing run environment for Pelican..."
```

Comment:
Rather than commenting this out, let's remove it. We try to avoid comment cruft floating around the codebase.